aws-rds-dbinstance/config/runtime.properties

```diff
@@ -0,0 +1,2 @@
+handler.backoff.delay=30000
+handler.backoff.timeout=129600000
```

Comment:
I believe this is quite hard to read, and unless you know this is in milliseconds. Can we utilize [ISO8601 durations](https://en.wikipedia.org/wiki/ISO_8601#Durations)?